apps/fe-admin/src/hooks/useDeferredValue.ts

```diff
@@ -0,0 +1,12 @@
+import { useEffect, useState } from 'react';
+
+export function useDeferredValue<T>(value: T, delay = 300): T {
```

Comment:
react has the same name for this hook https://react.dev/reference/react/useDeferredValue
try to use it